doc/mesh_filter/launch/test_gazebo.launch

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"  doc="If true, limits joint range [-PI, PI] on all joints." />
+  <arg name="paused" default="false" doc="Starts gazebo in paused mode" />
```

Comment:
These two parameters seem unnecessary for the tutorial, `gui` might make some sense to keep though.

Reply:
Removed them

Reply:
Removed